src/main/java/ru/skypro/homework/controller/CommentController.java

```diff
@@ -0,0 +1,77 @@
+package ru.skypro.homework.controller;
+
+
+import io.swagger.v3.oas.annotations.Operation;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.*;
+import ru.skypro.homework.dto.*;
+import ru.skypro.homework.service.CommentService;
+import ru.skypro.homework.service.impl.AdServiceImpl;
+
+@Slf4j
+@CrossOrigin(value = "http://localhost:3000")
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/ads/{adId}/comments")
+public class CommentController {
+
+    private final CommentService commentService;
+    private final AdServiceImpl adService;
+
+
+    @GetMapping
+    @Operation(summary = "Получение комментариев объявления", description = "getComments", tags = {"Комментарии"})
+    public ResponseEntity<CommentsDto> getComments(@PathVariable("id") Integer id,
```

Comment:
```suggestion
    public ResponseEntity<CommentsDto> getComments(@PathVariable("adId") Integer id,
```
Ошибка в имени параметра